src/app/docs/selection-box/selection-box-docs.component.html

```diff
@@ -0,0 +1,85 @@
+<sky-docs-demo-page
+  moduleName="SkySelectionBoxModule"
+  moduleSourceCodePath="src/app/public/modules/selection-box/"
+  pageTitle="Selection box"
+>
+  <sky-docs-demo-page-summary>
+    <!-- Page summary goes here -->
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel
+      (selectionChange)="onDemoSelectionChange($event)"
+    >
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-radio-group
+          heading="Form type"
+          initialValue="checkbox"
+          propertyName="mode"
+          [choices]="formChoices"
+        >
+        </sky-docs-demo-control-panel-radio-group>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+    <sky-selection-box-grid *ngIf="demoSettings?.mode === 'checkbox'">
+      <sky-selection-box *ngFor="let item of items"
+        [control]="checkbox"
```

Comment:
In this example, `checkbox` is a template reference variable that points to the element with `#checkbox` it comes into the component as whatever type of component it happens to be - in this case, a `SkyCheckboxComponent` component. Think of it like a pointer to another component. The docs are correct, but I'm open to suggestions if it can be more clear?

Reply:
That makes sense. I just wasn't understanding how this worked and then was confused by the links that don't point to anything. I think we might should clean up those links, but I'm going to resolve this for now and plan on circling back to that question later.